unitOneAssessmentPractice.js

```diff
@@ -4,41 +4,109 @@ let assert = require('assert')
 
 // Write a function called double that doubles a number
 
+const doubled = (num) => {
+  return num + num
+}
+
 // Uncomment out the next line to test your solution
 // runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
 
+const containsSeven = (num) => {
+  let stringedNum = num.toString()
+  for(let i =0; i < stringedNum.length; i++) {
+    if(stringedNum.includes(7)){
```

Comment:
This would get you the right answer, if the 7 was a string, there are a few things that aren't necessary. The for loop and the if statement aren't needed at all. after line 19, you can just `return stringedNum.includes('7')`

Reply:
This is because` includes` already loops through the string to check if the element exists in the str/array